### DISCUSSION TITLE
"Get Project as SB3" block

### DISCUSSION THREAD

**Discussion by snowboyz0825** [TurboWarp/extensions#1436] [Show and tell]
I didn't add this as an extension, as it's not, but I figured I'd show it off and put a place to download
![projectassb3](https://github.com/TurboWarp/extensions/assets/101284660/1b1b896c-cac3-4a0d-b43e-707f20ad0578)
This block uses a few extensions to let you get the SB3 of projects. You can (presumably download it) do whatever you want with it. 
[https://github.com/snowboyz0825/snowscodestuff/blob/main/Scratch%20Code/Get%20SB3.sprite3](https://github.com/snowboyz0825/snowscodestuff/blob/main/Scratch%20Code/Get%20SB3.sprite3) is the download link. 
![projectassb32](https://github.com/TurboWarp/extensions/assets/101284660/16a52985-0989-481b-bb7a-f52fa6339db0)
Here is the code if you just wanna see how it works without downloading. No clue what people will use this for, but have fun!

**Comment by BlueDome77** [comment]
The link is broken

**Reply by CST1229** [reply]
working link: https://github.com/snowboyz0825/snowscodestuff/blob/main/Scratch%20Code/Get%20SB3.sprite3

**Reply by snowboyz0825** [reply]
I have zero clue why it broke lol - nvm I put the display without the actual url

**Comment by AshimeeAlt** [comment]
this is cool
